components/MashDashboard/MashDashboard.spec.tsx

```diff
@@ -0,0 +1,90 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import {
+  mockedMashReferral,
+  mockedMashReferrals,
+} from 'factories/mashReferral';
+import { ReferralStage } from 'types';
+import MashDashboard from './MashDashboard';
+
+let tabType: string | undefined;
+let confirmationType: string | undefined;
+const mockedRouter = {
+  query: { confirmation: confirmationType, tab: tabType },
+  push: jest.fn(),
+};
+
+jest.mock('next/router', () => ({
+  useRouter: () => mockedRouter,
+}));
+
+describe('MashDashboard', () => {
+  const contactMashReferral = mockedMashReferral;
+  contactMashReferral.stage = ReferralStage.CONTACT;
```

Comment:
could we call the factory method with .build here?

Reply:
Done in 8b5297d

Reply:
Do we still need line 24 since we've called the build function with the stage we want above?

Reply:
Do we still need line 24 i.e `contactMashReferral.stage = ReferralStage.CONTACT;` since we've called the build function with the stage we want above?